views/components/ConfigDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import Select from '@material-ui/core/Select'
+import MenuItem from '@material-ui/core/MenuItem'
+import FormControl from '@material-ui/core/FormControl'
+import InputLabel from '@material-ui/core/InputLabel'
+
+const ConfigDropDown = ({
+  configurations,
+  updatePreferences,
+  currentConfig,
+  classes,
+}) => {
+  const [selectValue, setSelectValue] = useState('')
+  const handleConfigChange = (e) => {
+    setSelectValue(e.target.value.name)
+    updatePreferences(e.target.value)
+  }
+
+  useEffect(() => {
+    const configNameValue =
+      (!!configurations.length &&
+        configurations?.filter(
```

Comment:
Since we check that configurations responds to `.length` I think we can safely drop the optional chaining here
```suggestion
        configurations.filter(
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/129/commits/9023989bb9432f811f2a7bcd1fadedfa1e2ba2cb

Reply:
I don't see this file in that commit, was it changed as part of a different commit?

Reply:
This was changed entirely, removed the entire useEffect and used the currentPreference.config value instead.